<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="coatGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Powłoka</string>
    </property>
    <widget class="QCheckBox" name="coatCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>181</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Powłoka</string>
     </property>
    </widget>
    <widget class="QLabel" name="coatTypeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz rodzaj</string>
     </property>
    </widget>
    <widget class="QComboBox" name="coatTypeComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="coatPriceLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wprowadź stawkę</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="coatPriceTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="summaryGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>221</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Podsumowanie</string>
    </property>
    <widget class="QLabel" name="materialMassLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Masa materiału [kg]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="materialMassTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="coatCostTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="coatCostLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Koszt powłoki [zł]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="materialCostTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="materialCostLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Koszt materiału [zł]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="finalPriceTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="finalPriceLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cena końcowa [zł]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="finalPriceEuroTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="finalPriceEuroLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cena końcowa [euro]</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="paramsGroupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>221</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Parametry</string>
    </property>
    <widget class="QLabel" name="chooseMaterialShapeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz kształt </string>
     </property>
    </widget>
    <widget class="QComboBox" name="chooseMaterialShapeComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="chooseMaterialTypeComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="chooseMaterialTypeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz rodzaj</string>
     </property>
    </widget>
    <widget class="QComboBox" name="chooseMaterialGenreComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="chooseMaterialGenreLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz gatunek</string>
     </property>
    </widget>
    <widget class="QLabel" name="dimension1Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wymiar 1 [mm]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dimension1TextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dimension2Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wymiar 2 [mm]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dimension2TextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dimension3Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wymiar 3 [mm]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dimension3TextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dimension4Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wymiar 4 [mm]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dimension4TextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="materialPriceLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wprowadź cenę [zł/kg]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="materialPriceTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
